x/nft/client/cli/cli_test.go

```diff
@@ -1,397 +0,0 @@
-package cli_test
-
-import (
-	"fmt"
-	"testing"
-
-	"github.com/gogo/protobuf/proto"
-	"github.com/stretchr/testify/suite"
-	"github.com/tidwall/gjson"
```

Comment:
I think that's the only place where a library from `tidwall` was used, so could remove that from app.go and app.mod
There seem to be some other unused libraries as well so a cleanup would be good.